idea64.vmoptions

```diff
@@ -0,0 +1,20 @@
+-Xms128m
+-Xmx750m
+-XX:ReservedCodeCacheSize=240m
+-XX:+UseConcMarkSweepGC
+-XX:SoftRefLRUPolicyMSPerMB=50
+-ea
+-XX:CICompilerCount=2
+-Dsun.io.useCanonPrefixCache=false
+-Djdk.http.auth.tunneling.disabledSchemes=""
+-XX:+HeapDumpOnOutOfMemoryError
+-XX:-OmitStackTraceInFastThrow
+-Djdk.attach.allowAttachSelf=true
+-Dkotlinx.coroutines.debug=off
+-Djdk.module.illegalAccess.silent=true
+-Dawt.useSystemAAFontSettings=lcd
+-Dsun.java2d.renderer=sun.java2d.marlin.MarlinRenderingEngine
+-Dsun.tools.attach.tmp.only=true
+
+-Djava.io.tmpdir=/var/cache/tmp
```

Comment:
I see that this file is quite different from the one provided by JetBrains. Are you entirely sure you need all those additional settings?

Another approach would be to just augment the default one with the new settings you are sure you are going to absolutely need because of the Flatpak business.

Reply:
I had just copied the one from IntelliJ and added that last line for java tmpdir. I think on latest versions of IntelliJ the file changed quite a bit and I forgot to update it. I'll commit now the same file with just last line added.

Reply:
What I was trying to say is that updates for this Flatpak could be automated with the `x-checker` thing. If the lines that are needed are just appended to `idea64.vmoptions`, we can make sure than every time the JetBrains folks update that file, the Flatpak would have those changes plus the ones you would like to introduce.

The only flaw would be if we are overwriting something from the original one, which is unlikely.

Reply:
Sorry, I don't know about this option. Can you pointing me to an example or propose your solution here?
We could even merge this and then you could create another PR against beta to improve this part.

Reply:
That's fine, it could be done later. I just don't use this edition, but I will those later.

Do you want me to give it a whirl to the latest build or are you comfortable with the changes? — as in nothing super-obvious is failing so far. If so, let's merge and improve it later on.

Reply:
It's working fine for me, but if you have some time, please give it a try first and then we can merge it.
Also, shall we do a squash commit to keep history cleaner? Up to you.